frontend/src/components/admin/AdminDashboard.js

```diff
@@ -13,4 +14,4 @@ export const AdminDashboard = () => (
   </div>
 );
 
-export default AdminDashboard;
+export default AuthorizationGuard("role_admin")(AdminDashboard);
```

Comment:
Do we want to put these auth keys into constants?

Reply:
updated!